src/test/java/domain/WinningLottoTest.java

```diff
@@ -4,94 +4,51 @@
 
 class WinningLottoTest {
 
     @Test
     void 보너스볼과_당첨번호가_중복된_경우_예외를_반환한다() {
         //given
         Number bonus = new Number(3);
-        Lotto lotto = new Lotto(
-                List.of(
-                        new Number(1),
-                        new Number(2),
-                        new Number(3),
-                        new Number(4),
-                        new Number(5),
-                        new Number(6)
-                )
-        );
+        Lotto lotto = Lotto.createWinningLotto(List.of(1, 2, 3, 4, 5, 6));
 
-        //when //then
+        //when & then
         assertThatThrownBy(() -> new WinningLotto(lotto, bonus))
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessage("보너스 볼 번호는 당첨 번호와 중복될 수 없습니다.");
     }
 
-    @Test
-    void 당첨_번호와_몇개가_같은지_계산할_수_있다() {
+    @ParameterizedTest
+    @MethodSource("generateLottos")
+    void 당첨순위를_판단할_수_있다(List<Integer> numbers, Rank expected) {
         //given
         Number bonus = new Number(7);
-        Lotto lotto = new Lotto(
-                List.of(
-                        new Number(1),
-                        new Number(2),
-                        new Number(3),
-                        new Number(4),
-                        new Number(5),
-                        new Number(6)
-                )
-        );
+        Lotto lotto = Lotto.createWinningLotto(List.of(1, 2, 3, 4, 5, 6));
         WinningLotto winningLotto = new WinningLotto(lotto, bonus);
-        Lotto purchasedLotto = new Lotto(
-                List.of(
-                        new Number(1),
-                        new Number(2),
-                        new Number(10),
-                        new Number(11),
-                        new Number(12),
-                        new Number(13)
-                )
-        );
-
+        Lotto purchasedLotto = Lotto.createRandomLotto((int size) -> numbers);
         //when
-        int matchCount = winningLotto.calculateMatchCount(purchasedLotto);
-
+        Rank actual = winningLotto.calculateRank(purchasedLotto);
         //then
-        assertThat(matchCount).isEqualTo(2);
+        assertThat(expected).isEqualTo(actual);
     }
 
-
-    @Test
-    void 보너스번호와_일치하는지_판단한다() {
-        //given
-        Number bonus = new Number(7);
-        Lotto lotto1 = new Lotto(
-                List.of(
-                        new Number(1),
-                        new Number(2),
-                        new Number(3),
-                        new Number(4),
-                        new Number(5),
-                        new Number(6)
-                )
+    private static Stream<Arguments> generateLottos() {
+        return Stream.of(
+                Arguments.of(List.of(1, 2, 3, 4, 5, 6), Rank.FIRST
+                ),
+                Arguments.of(List.of(1, 2, 3, 4, 5, 7), Rank.SECOND
+                ),
+                Arguments.of(List.of(1, 2, 3, 4, 5, 8), Rank.THIRD
+                ),
```

Comment:
```suggestion
                Arguments.of(List.of(1, 2, 3, 4, 5, 6), Rank.FIRST),
                Arguments.of(List.of(1, 2, 3, 4, 5, 7), Rank.SECOND),
                Arguments.of(List.of(1, 2, 3, 4, 5, 8), Rank.THIRD),
```
로 줄바꿈이 없어도 될 것 같아요 :)